### ISSUE TITLE
memory editor GUI improvments

### ISSUE THREAD

**Issue by sqrt0x** [scanmem/scanmem#447]
First of all, I’m a big fan of the Scanmem project and want to thank you for this masterpiece.

I'm here to talk about this https://github.com/scanmem/scanmem/issues/357
this ticket was opened in 2019 and has not been closed since so I’m curious if there have been any updates.

Is there any built-in feature/plugin I'm unaware of, or must I change the code and do a pull request?

### Goals:
- ~~Display every 4 bytes with different colors~~
- Display  memory as float/int values instead of hexadecimal ones

**Comment by sqrt0x** [comment]
I can make the code cleaner and submit this as a pull request <br><br>

![image](https://github.com/user-attachments/assets/789973ae-662d-4b4e-b584-4bd8cde8c2db)

**Comment by sriemer** [comment]
Sounds good. Thanks for the PR. Usually I only use the CLI tool but stopped this in 2019 when my health collapsed. You can imagine that me and 12345ieee have different priorities now. We are looking for new maintainers.
Thanks so much for starting with thanks. 💓 We put so much effort, hard work, and our leisure/sleep time into this project.
But now it is time for us to pass this over. Would you be interested? TIA

**Comment by 12345ieee** [comment]
Definitely, @sriemer can you add a big note to the readme that we're looking for people to take over?
Maybe also open an issue we can link to.

I know the folks at PINCE ( @korcankaraokcu ) might be interested, also I see a couple forks with recent significant work, like @OpenA 's and @sivertac 's.

**Comment by FatSquare** [comment]
Hello,  thanks for you offer of course I'll be more than happy and ready to take over and contribute to this project. I'm using scanmem daily and I don't want to see it going forever. Either way, I'm thankful for what you guys have done for this project so far. Thanks again
